primitives/src/opcodes.rs

```diff
@@ -188,15 +188,15 @@ all_opcodes! {
     OP_2DROP => 0x6d, "Drops the top two stack items.";
     OP_2DUP => 0x6e, "Duplicates the top two stack items as AB -> ABAB.";
     OP_3DUP => 0x6f, "Duplicates the two three stack items as ABC -> ABCABC.";
-    OP_2OVER => 0x70, "Copies the two stack items of items two spaces back to the front, as xxAB -> ABxxAB.";
-    OP_2ROT => 0x71, "Moves the two stack items four spaces back to the front, as xxxxAB -> ABxxxx.";
+    OP_2OVER => 0x70, "Copies the two stack items of items two spaces back to the front, as xxAB -> `ABxxAB`.";
```

Comment:
In 656b2ce65e260f67302ceb63a061d1c7b2ab2a71:

Why add backticks around `ABxxAB` but not `xxAB`?

Reply:
After reading https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown I think the lint identifies `ABxxAB` because it's camel case while `xxAB` is not camel case.  As I understand, the lint looks for things that could be code pieces in docs by looking for things that are camel case without backticks.

I created this PR to get the lint to pass although clearly there are a lot of other things that could be noted as code elements using backticks.

Reply:
This lint seems pretty silly to me, although I guess it doesn't hurt to backtick more things.

Reply:
This requirement is a bit ugly but better than my previous thought to allow the lint. Lets just back tick both.

```rust
OP_2OVER => 0x70, "Copies the two stack items of items two spaces back to the front, as `xxAB` -> `ABxxAB`.";
```

Reply:
They are already both have back ticks in this PR